### PR TITLE
Add warnings for the resetall command and correct the text in the "Error while loading" screen

### PR DESCRIPTION
**Pull request by MattMayuga**

### CODE REVIEW

File: parts/language/lang_en.lua [hunk 1]
```diff
@@ -872,7 +872,7 @@ return{
         "You can navigate the menu with a keyboard, but only in this screen.",
         "You can open the save directory from the Stats page.",
         "You can perform a spin with 28 of the 29 minoes in this game; the exception being O1.",
-        "You can set the spawning orientation for each tetromino.",
+        "You can set the spawning orientation for each piece.",
```
Comment (user670) [inline parts/language/lang_en.lua:875]: Only tetrominoes can be changed, do-, tri-, pentominoes can't (yet)

File: parts/scenes/app_console.lua [hunk 2]
```diff
@@ -579,7 +579,7 @@ local commands={}do
             {
                 code="dtw",
                 scene='app_dtw',
-                description="Don't Touch White, a.k.a. Piano Tiles"
+                description="Don't Touch White (Piano Tiles)"
```
Comment (user670) [inline parts/scenes/app_console.lua:582]: You missed one on line 607

File: parts/scenes/app_console.lua [hunk 3]
```diff
@@ -667,36 +672,44 @@ local commands={}do
...
+                        log{C.R,"Deleting all data in 3..."}SFX.play('ready')SFX.play('clear_2')for _=1,60 do coroutine.yield()end
+                        log{C.R,"Deleting all data in 2..."}SFX.play('ready')SFX.play('clear_3')for _=1,60 do coroutine.yield()end
+                        log{C.R,"Deleting all data in 1..."}SFX.play('ready')SFX.play('clear_3')for _=1,60 do coroutine.yield()end
+                        log{C.R,"Deleting all data in 0..."}SFX.play('start')SFX.play('clear_4')for _=1,60 do coroutine.yield()end
                         outputBox.hide=true for _=1,26 do coroutine.yield()end
                         FILE.clear_s('')love.event.quit()
                     end)
                 end}
-                button:setObject("Techmino is fun. Bye.")
+                button:setObject("The saved data was erased!")
```
Comment (MrZ626) [inline parts/scenes/app_console.lua:690]: This line is the text on the button, no need to change it

Comment (MrZ626) [reply parts/scenes/app_console.lua:690]: And I think add the 'yes' is not neccessary, normally you need input `resetall` for three times

**Pull request merged**